### PR TITLE
[zh-cn] Translate /docs/concepts/security/api-server-bypass-risks.md to Chinese

### PR DESCRIPTION
**Pull request by ktKongTong**
fixed: https://github.com/kubernetes/website/issues/37689

Translate Chinese version for api-server-bypass-risks.md

[Preview](https://deploy-preview-37724--kubernetes-io-main-staging.netlify.app/zh-cn/docs/concepts/security/api-server-bypass-risks/)

### CODE REVIEW

File: content/zh-cn/docs/concepts/security/api-server-bypass-risks.md [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+(users and services) interacting with it. 
+ -->
+Kubernetes API 服务器是外部（用户和服务）与集群交互的主要入口。
+
+<!-- 
+As part of this role, the API server has several key built-in security controls, such as
+audit logging and {{< glossary_tooltip text="admission controllers" term_id="admission-controller" >}}. However, there are ways to modify the configuration
+or content of the cluster that bypass these controls.
+ -->
+作为此角色的一部分，API 服务器有几个关键的内置安全控制，例如审计日志和{{< glossary_tooltip text="准入控制器" term_id="admission-controller" >}}。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/security/api-server-bypass-risks.md:29]:
```suggestion
作为此角色的一部分，API 服务器有几个关键的内置安全控制，
例如审计日志和{{< glossary_tooltip text="准入控制器" term_id="admission-controller" >}}。
```

File: content/zh-cn/docs/concepts/security/api-server-bypass-risks.md [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+但是，有一些方法可以绕过这些安全控制从而修改集群的配置或内容。
+
+<!-- 
+This page describes the ways in which the security controls built into the
+Kubernetes API server can be bypassed, so that cluster operators
+and security architects can ensure that these bypasses are appropriately restricted.
+ -->
+本页描述了绕过 Kubernetes API 服务器中内置安全控制的几种方式，以便集群运维人员和安全架构师可以确保这些绕过方式被适当地限制。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/security/api-server-bypass-risks.md:37]:
```suggestion
本页描述了绕过 Kubernetes API 服务器中内置安全控制的几种方式，
以便集群运维人员和安全架构师可以确保这些绕过方式被适当地限制。
```

File: content/zh-cn/docs/concepts/security/api-server-bypass-risks.md [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+
+<!-- 
+The {{< glossary_tooltip text="kubelet" term_id="kubelet" >}} on each node loads and
+directly manages any manifests that are stored in a named directory or fetched from
+a specific URL as [*static Pods*](/docs/tasks/configure-pod-container/static-pod) in
+your cluster. The API server doesn't manage these static Pods. An attacker with write
+access to this location could modify the configuration of static pods loaded from that
+source, or could introduce new static Pods.
+ -->
+每个节点上的 {{< glossary_tooltip text="kubelet" term_id="kubelet" >}} 会加载并直接管理集群中存储在指定目录中或从特定 URL 获取的[**静态 Pod**](/zh-cn/docs/tasks/configure-pod-container/static-pod) 清单。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/security/api-server-bypass-risks.md:53]:
```suggestion
每个节点上的 {{< glossary_tooltip text="kubelet" term_id="kubelet" >}}
会加载并直接管理集群中存储在指定目录中或从特定 URL
获取的[**静态 Pod**](/zh-cn/docs/tasks/configure-pod-container/static-pod) 清单。
```

File: content/zh-cn/docs/concepts/security/api-server-bypass-risks.md [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+ -->
+## kubelet API {#kubelet-api}
+
+<!-- 
+The kubelet provides an HTTP API that is typically exposed on TCP port 10250 on cluster
+worker nodes. The API might also be exposed on control plane nodes depending on the Kubernetes
+distribution in use. Direct access to the API allows for disclosure of information about
+the pods running on a node, the logs from those pods, and execution of commands in
+every container running on the node.
+ -->
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/security/api-server-bypass-risks.md:114]:
```suggestion
-->
```

File: content/zh-cn/docs/concepts/security/api-server-bypass-risks.md [hunk 5]
```diff
@@ -0,0 +1,275 @@
...
+kubelet 提供了一个 HTTP API，通常暴露在集群工作节点上的 TCP 端口 10250 上。
+在某些 Kubernetes 发行版中，API 也可能暴露在控制平面节点上。
+对 API 的直接访问允许公开有关运行在节点上的 Pod、这些 Pod 的日志以及在节点上运行的每个容器中执行命令的信息。
+
+<!-- 
+When Kubernetes cluster users have RBAC access to `Node` object sub-resources, that access
+serves as authorization to interact with the kubelet API. The exact access depends on
+which sub-resource access has been granted, as detailed in [kubelet authorization](https://kubernetes.io/docs/reference/access-authn-authz/kubelet-authn-authz/#kubelet-authorization).
+ -->
+当 Kubernetes 集群用户具有对 `节点` 对象子资源 RBAC 访问权限时，该访问权限可用作与 kubelet API 交互的授权。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/security/api-server-bypass-risks.md:124]:
```suggestion
当 Kubernetes 集群用户具有对 `Node` 对象子资源 RBAC 访问权限时，该访问权限可用作与 kubelet API 交互的授权。
```
`Node` 应该不用翻，下文也有几处。

File: content/zh-cn/docs/concepts/security/api-server-bypass-risks.md [hunk 6]
```diff
@@ -0,0 +1,275 @@
...
+实际的访问权限取决于授予了哪些子资源访问权限，详见 [kubelet 鉴权](https://kubernetes.io/zh-cn/docs/reference/access-authn-authz/kubelet-authn-authz/#kubelet-authorization)。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/security/api-server-bypass-risks.md:125]:
```suggestion
实际的访问权限取决于授予了哪些子资源访问权限，详见
[kubelet 鉴权](/zh-cn/docs/reference/access-authn-authz/kubelet-authn-authz/#kubelet-authorization)。
```

File: content/zh-cn/docs/concepts/security/api-server-bypass-risks.md [hunk 7]
```diff
@@ -0,0 +1,275 @@
...
+<!-- 
+Direct access to etcd is not subject to Kubernetes admission control and is not logged
+by Kubernetes audit logging. An attacker who has read access to the API server's
+etcd client certificate private key (or can create a new trusted client certificate) can gain
+cluster admin rights by accessing cluster secrets or modifying access rules. Even without
+elevating their Kubernetes RBAC privileges, an attacker who can modify etcd can retrieve any API object
+or create new workloads inside the cluster.
+ -->
+对 etcd 的直接访问不受 Kubernetes 准入控制的影响，也不会被 Kubernetes 审计日志记录。
+具有对 API 服务器的 etcd 客户端证书私钥的读取访问权限（或可以创建一个新的受信任的客户端证书）的攻击者可以通过访问集群 Secret 或修改访问规则来获得集群管理员权限。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/security/api-server-bypass-risks.md:197]:
```suggestion
具有对 API 服务器的 etcd 客户端证书私钥的读取访问权限（或可以创建一个新的受信任的客户端证书）
的攻击者可以通过访问集群 Secret 或修改访问规则来获得集群管理员权限。
```

**Review by windsonsea — COMMENTED** [review]
长句可以适当折行

File: content/zh-cn/docs/concepts/security/api-server-bypass-risks.md [hunk 8]
```diff
@@ -0,0 +1,280 @@
...
+## 容器运行时套接字 {#runtime-socket}
+
+<!-- 
+On each node in a Kubernetes cluster, access to interact with containers is controlled 
+by the container runtime (or runtimes, if you have configured more than one). Typically,
+the container runtime exposes a Unix socket that the kubelet can access. An attacker with
+access to this socket can launch new containers or interact with running containers.
+ -->
+在 Kubernetes 集群中的每个节点上，与容器交互的访问都由容器运行时控制。
+通常，容器运行时会公开一个 kubelet 可以访问的 Unix 套接字。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/security/api-server-bypass-risks.md:248]:
```suggestion
通常，容器运行时会公开一个 kubelet 可以访问的 UNIX 套接字。
```

File: content/zh-cn/docs/concepts/security/api-server-bypass-risks.md [hunk 9]
```diff
@@ -0,0 +1,281 @@
...
+The default anonymous access doesn't make this assertion with the control plane.
+ -->
+kubelet API 可以配置为以多种方式验证请求。
+默认情况下，kubelet 的配置允许匿名访问。大多数 Kubernetes 提供商将默认值更改为使用 Webhook 和证书身份认证。
+这使得控制平面能够确保调用者访问 `Node` API 资源或子资源是经过授权的。但控制平面不能确保默认的匿名访问也是如此。
+
+<!-- 
+### Mitigations
+ -->
+### 缓解措施
```
Comment (Sea-n) [inline content/zh-cn/docs/concepts/security/api-server-bypass-risks.md:154]:
```suggestion
### 缓解措施   {#mitigations}
```

File: content/zh-cn/docs/concepts/security/api-server-bypass-risks.md [hunk 10]
```diff
@@ -0,0 +1,281 @@
...
+静态 Pod 被限制访问 Kubernetes API 中的其他对象。例如，你不能将静态 Pod 配置为从集群挂载 Secret。
+但是，这些 Pod 可以执行其他安全敏感的操作，例如挂载来自下层节点的 `hostPath` 卷。
+
+<!-- 
+By default, the kubelet creates a {{< glossary_tooltip text="mirror pod" term_id="mirror-pod">}}
+so that the static Pods are visible in the Kubernetes API. However, if the attacker uses an invalid
+namespace name when creating the Pod, it will not be visible in the Kubernetes API and can only
+be discovered by tooling that has access to the affected host(s).
+ -->
+默认情况下，kubelet 会创建一个 {{< glossary_tooltip text="镜像 Pod（Mirror Pod）" term_id="mirror-pod">}}，
```
[the remark this answers is not in the document]
Comment (ktKongTong) [reply content/zh-cn/docs/concepts/security/api-server-bypass-risks.md:75]: It's begin with a Chinese character. why need a whitespace here? And there is already a visible space even if the space is not there. What do you think? @Sea-n
image

**Comment by windsonsea** [comment]
The preview looks good to me.
/lgtm

**Pull request merged**